### ISSUE TITLE
a way to get only the changed objects from a dataset

### ISSUE THREAD

**Issue by balugaa** [handsontable/handsontable#1407]
Hi, 

Is it possible to get only the changed objects from handsontable in order to send back to the server for persistence, currently i it appears the only way of doing this would be using afterchange and collecting the row numbers, and then when users click save i create a new array with the changed objects based on the row numbers i had collected from the after change to the server, 

Posting back on each after change event is not really an option i.e. i cannot implement an autosave type function, as for this project would not be advised. 

basically what i want to do is  load the data from ajax via get, and post only the changes if any back instead of the whole dataset again .. 

Not sure if there is an easier way to do this already but it would be very handy

**Comment by AMBudnik** [comment]
Issue closed in lack of other unsolved questions.
